tests/integration-tests/tests/storage/storage_common.py

```diff
@@ -458,58 +485,98 @@ def _test_data_repository_task(remote_command_executor, mount_dir, bucket_name,
     assert_that(file_permissions).is_equal_to("0100777")
 
 
-def _test_export_path(remote_command_executor, mount_dir, bucket_name, region):
+def _test_export_path(remote_command_executor, mount_dir, bucket_name, region, file_cache_path=None):
     logging.info("Testing fsx lustre export path")
-    remote_command_executor.run_remote_command(
-        "echo 'Exported by FSx Lustre' > {mount_dir}/file_to_export".format(mount_dir=mount_dir)
-    )
-    remote_command_executor.run_remote_command(
-        "sudo lfs hsm_archive {mount_dir}/file_to_export && sleep 5".format(mount_dir=mount_dir)
-    )
-    remote_command_executor.run_remote_command(
-        "sudo aws s3 cp --region {region} s3://{bucket_name}/export_dir/file_to_export ./file_to_export".format(
-            region=region, bucket_name=bucket_name
+    if file_cache_path:
+        remote_command_executor.run_remote_command(
+            "echo 'Exported by FSx Lustre' | sudo tee {mount_dir}{file_cache_path}file_to_export".format(
+                mount_dir=mount_dir, file_cache_path=file_cache_path
+            )
+        )
+        remote_command_executor.run_remote_command(
+            "sudo lfs hsm_archive {mount_dir}{file_cache_path}file_to_export && sleep 5".format(
+                mount_dir=mount_dir, file_cache_path=file_cache_path
+            )
+        )
+        export_result = remote_command_executor.run_remote_command(
+            "sudo lfs hsm_state {mount_dir}{file_cache_path}file_to_export".format(
+                mount_dir=mount_dir, file_cache_path=file_cache_path
+            )
+        )
+        assert_that(export_result.stdout).is_equal_to(
+            "{mount_dir}{file_cache_path}file_to_export: (0x00000009) exists archived, archive_id:1".format(
+                mount_dir=mount_dir, file_cache_path=file_cache_path
+            )
+        )
+
+        remote_command_executor.run_remote_command(
+            "sudo aws s3 cp --region {region} s3://{bucket_name}/file_to_export ./file_to_export".format(
```

Comment:
Why is `sudo` required?

Reply:
The user in different OS does not have access to upload the files. So we use sudo